__tests__/utils.ts

```diff
@@ -0,0 +1,227 @@
+import { createMocks, RequestMethod } from "node-mocks-http";
+import { TaxForOrderRes } from "taxjar/dist/types/returnTypes";
+import {
+  CheckoutPayload,
+  FetchTaxesPayload,
+  ResponseTaxPayload,
+} from "../backend/types";
+
+export const mockRequest = (
+  method: RequestMethod = "GET",
+  event?: string,
+  domain?: string,
+  signature?: string
+) => {
+  const { req, res } = createMocks({ method });
+  req.headers = {
+    "Content-Type": "application/json",
+  };
+  if (domain !== undefined) {
+    req.headers["saleor-domain"] = domain;
+  }
+  if (event !== undefined) {
+    req.headers["saleor-event"] = event;
+  }
+  if (signature !== undefined) {
+    req.headers["x-saleor-signature"] = signature;
+  }
+
+  return { req, res };
+};
+
+export const getDummyFetchTaxesPayload = () => {
```

Comment:
For all of these `getDummy__` if you're returning static data just export it as a constant instead of a function:
```suggestion
export const dummyFetchTaxesPayload: FetchTaxesPayload = { ... }
```